module/VuFind/src/VuFind/Mailer/Mailer.php

```diff
@@ -231,6 +288,15 @@ public function send($to, $from, $subject, $body, $cc = null, $replyTo = null)
             if ($replyTo) {
                 $message->addReplyTo($replyTo);
             }
+
+            $headers = $message->getHeaders();
+            if (!$headers->has('Content-Type')) {
+                $ctype = new ContentType();
+                $ctype->setType('text/plain');
```

Comment:
Would it make sense to use Mime::TYPE_TEXT here for consistency with the instances above?

Reply:
Sorry for the late reply. Yes, You're right. Shall I do something about it?

Reply:
If you have a moment and don't mind making the edit, please feel free... but if not, I can take care of it when I update this pull request for compatibility with the current master branch.

Reply:
How can I access this PR, the original had been closed? Or should I make up a second one from finc?

Reply:
Oh, my apologies, I forgot that I had already opened an updated version of the PR! I'll take care of the two edits and then merge it on my end. I could also give you write access to the branch underlying the pull request so you could do it yourself, but given that both of these changes are simple, I don't need to make you do the work for me. Thanks again for your help with these improvements!

Reply:
Thank You too for Your replies and community tutorial yesterday.